framework/src/modules/auth/schemas.ts

```diff
@@ -326,3 +326,15 @@ export const sortMultisignatureGroupResponseSchema = {
 	},
 	required: ['mandatoryKeys', 'optionalKeys', 'signatures'],
 };
+
+export const multiSigRegMsgTagSchema = {
+	$id: '/auth/mutliSignatureRegistrationSignatureMessageTagResponse',
+	type: 'object',
+	properties: {
+		tag: {
+			type: 'string',
+			fieldNumber: 1,
+		},
```

Comment:
```suggestion
		tag: {
			type: 'string',
		},
```
fieldNumber not needed since it wont be encoded